### ISSUE TITLE
boats being buggy

### ISSUE THREAD

**Issue by aicelo** [Uuvana-Studios/longvinter-windows-client#1281]
not too often, but sometimes when i get into a boat, a little red maker pops up on my screen and if i attempt to use the boat (move forward, back or side to side) it like teleports me to another boat.

doesnt happen too often but annoying to say the least

**Comment by esme82** [comment]
Yes, it's happened to me. It's the boathouse near Filemon's and when I try to wiggle the wooden boat it teleports me to Diogo's.
